webapp/src/components/gallery/galleryCard.tsx

```diff
@@ -93,6 +96,16 @@ const GalleryCard = React.memo((props: Props) => {
                                 mutator.duplicateCard(card.id)
                             }}
                         />
+                        <Menu.Text
+                            icon={<LinkIcon/>}
+                            id='copy'
+                            name={intl.formatMessage({id: 'GalleryCard.copyLink', defaultMessage: 'Copy link'})}
+                            onClick={() => {
+                                const url = Utils.addQueryParamToCurrentURL('c', cardTree.card.id)
```

Comment:
When this [PR](https://github.com/mattermost/focalboard/pull/1050) goes in, this line should no longer be necessary.